jina/orchestrate/deployments/__init__.py

```diff
@@ -471,6 +478,13 @@ def __init__(
             args = ArgNamespace.kwargs2namespace(kwargs, parser, True)
         self.args = args
         self._gateway_load_balancer = False
+        if self.args.provider == ProviderType.SAGEMAKER:
+            if self.args.port != 8080:
```

Comment:
please check this also works when I pass 8080, I am not sure if the port here is a list or a single int, check that

Reply:
I have changed it to - if Deployment uses 8080 with provider=sagemaker, it'd raise `ValueError` and a warning before changing the Executor pod's port to 8080.